docs/ui/AppearanceTable.tsx

```diff
@@ -19,58 +19,48 @@ export const AppearanceTable = ({ component }: AppearanceTableProps) => {
 
   const appearances = getAppearance(component, themes[current]);
 
+  if (appearances?.variant?.length === 0 && appearances?.size?.length === 0) {
```

Comment:
Can't we use now the emptyState prop instead of this?

Reply:
yes, this is what I plan to do in a separate PR! :) 

(this is actually why I wanted this feature ^^)